### DISCUSSION TITLE
I won a block and didn't receive the reward

### DISCUSSION THREAD

**Discussion by Serhii-21** [Chia-Network/chia-blockchain#8581] [Support]
Hello
I won the block  # 748254 in space pool and there is no revenue yet
21.08.2021, 22:34
Hash ba0d4ea02f1968b6025c40417e20f651b715e29b8a36ffad9ccb5d2db647c2fb
I checked my Farmer Rewards Target Address by chia faucet. Everything fine
Farmer Rewards Address and Pool Reward Address
xch1pklkn2s52s424fdchg6vpje84xv8mn6x4n5l0fl8svux27pwdevqfpy5t4
Help me please

**Comment by loppefaaret** [comment]
748254 was won by a farmer that had "farmer rewards target address" set to `xch1ahc9q0lyy0h0m7ylzagx2uqn0zqhhkul6augvluuggk4le444trsepn7hs`
if your farmer has it set to `xch1pklkn2s52s424fdchg6vpje84xv8mn6x4n5l0fl8svux27pwdevqfpy5t4` - you did not win 748254
you are not the first from pool.space to ask about this issue... several other users have had this question from farming on space pool

**Comment by Serhii-21** [comment]
So, then I don't undestand who stole my revenue, pool?
On the spase pool web site, they explain like: The 0.25xch portion of the block win comes directly from the block-chain — the pool never sees this!
Here is my screenshot with the block number
![Chia_Block](https://user-images.githubusercontent.com/91313622/135244583-2fde36b5-2a40-4552-a7fb-eca52dcf5483.png)

**Comment by emlowe** [comment]
There are various clients which change this address - software from corepool, Hpool, and maybe poolharvest can do this.

This is a good blog post by spacepool: https://blog.pool.space/how-to-find-your-0-25xch-after-winning-a-block-in-the-pool-fc61d0bee5c5

**Comment by Serhii-21** [comment]
Thank you, I understand that. But How to check if my pay out address was changed via another software? The Farmer Rewards Target Address in Chia GUI is mine, I checked it by chia faucet.
And how to explain another moment, I recently won my second block and I received my reward. At the same time I didn't change any settings and addresses since I won my first block
![Pool_2XCH](https://user-images.githubusercontent.com/91313622/136330226-26acb6c8-facd-48ae-b1d8-6ebbe1168652.png)
![0 25XCH-SEC-2021-10-04](https://user-images.githubusercontent.com/91313622/136330362-da395843-70b0-4540-9e3b-2d735608deee.png)

**Comment by plot14** [comment]
have the same problem , lost my won  . can someone help us ? or must we leave the progect ?
